engine/launcher/src/main/scala/org/enso/launcher/cli/GlobalCLIOptions.scala

```diff
@@ -15,30 +14,27 @@ import org.enso.loggingservice.{ColorMode, LogLevel}
   *                     printed
   * @param useJSON specifies if output should be in JSON format, if it is
   *                supported (currently only the version command supports JSON)
-  * @param colorMode specifies if console output should contain colors
   * @param internalOptions options that are remembered to pass them to launcher
   *                        child processes
   */
 case class GlobalCLIOptions(
   autoConfirm: Boolean,
   hideProgress: Boolean,
   useJSON: Boolean,
-  colorMode: ColorMode,
```

Comment:
I assume `colorMode` was only used in logging-related settings and it is no longer used?

What is the new behaviour of terminal colors? Is the logger output never colored now?

Reply:
Coloring is not supported